src/containers/Search/ActionBarContainer.tsx

```diff
@@ -41,8 +42,22 @@ const {
 const STAGE_IPFS_HASH = 3.1;
 const STAGE_KEPLR_APPROVE = 3.2;
 
-class ActionBarContainer extends Component {
-  constructor(props) {
+// generated
+interface Props {
+  defaultAccount: DefaultAccount;
+
+  textBtn?: string;
+  placeholder?: string;
+  rankLink?: string;
+  update: () => void;
+  signer: any;
+  node: any;
+  signingClient: any;
+  keywordHash: string;
+}
+
+class ActionBarContainer extends Component<Props, any> {
```

Comment:
need to be refactored to functional component

Reply:
it is risky to refactor large module that is working.
when modify such modules, possible to refactor